datafusion/sqllogictest/test_files/ddl.slt

```diff
@@ -775,3 +775,9 @@ physical_plan CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/te
 
 statement ok
 drop table t;
+
+statement ok
+set datafusion.explain.logical_plan_only=true;
+
+statement error DataFusion error: This feature is not implemented: Temporary views not supported
+explain CREATE TEMPORARY VIEW y AS VALUES (1,2,3);
```

Comment:
This result of explain don't seem right. This PR should support the logical plan for TEMPORARY DDL.